integration/clean/clean_aws_resource.go

```diff
@@ -0,0 +1,145 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: MIT
+
+//go:build clean
+// +build clean
+
+package main
+
+import (
+	"context"
+	"log"
+	"time"
+
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/config"
+	"github.com/aws/aws-sdk-go-v2/service/ec2"
+	ec2Type "github.com/aws/aws-sdk-go-v2/service/ec2/types"
+	"github.com/aws/aws-sdk-go-v2/service/ssm"
+	ssmType "github.com/aws/aws-sdk-go-v2/service/ssm/types"
+	smithyTime "github.com/aws/smithy-go/time"
+)
+
+const (
+	daysToKeep   = 30
```

Comment:
What is the reason for going from 60 to 30? I'm mostly fine with it. I just wanted to understand why the change.

Reply:
Every time we do a release, it would rather spin up the new AMI, relevant resources,... However, these release could mostly takes up to 2 - 5 weeks. Therefore, reduce to delete resources more frequently.